hazelcast/src/main/java/com/hazelcast/internal/bootstrap/TpcServerBootstrap.java

```diff
@@ -70,8 +72,7 @@ public TpcServerBootstrap(NodeEngineImpl nodeEngine) {
         this.nodeEngine = nodeEngine;
         this.logger = nodeEngine.getLogger(TpcServerBootstrap.class);
         this.ss = (InternalSerializationService) nodeEngine.getSerializationService();
-        this.enabled = Boolean.parseBoolean(getProperty("hazelcast.tpc.enabled", "false"))
-                || nodeEngine.getConfig().getAltoConfig().isEnabled();
+        this.enabled = nodeEngine.getConfig().getAltoConfig().isEnabled();
```

Comment:
It is key that it can be configured using System properties. So probably this isn't what we want.

Reply:
https://github.com/hazelcast/hazelcast/pull/23319#discussion_r1064336149

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/009609eec0df96ba2a9327217aa40ce9d50c417b